s3fs/core.py

```diff
@@ -142,7 +169,10 @@ def _ls(self, path, refresh=False):
                     f['Size'] = 0
                     del f['Name']
             else:
-                files = self.s3.list_objects(Bucket=bucket).get('Contents', [])
+                try:
+                    files = self.s3.list_objects(Bucket=bucket).get('Contents', [])
+                except ClientError:
+                    files = []
```

Comment:
Would it be fair to say that this is an enhancement unrelated to the PR, so should merge and make a new PR that does this well? PR #1 isn't even in yet.